src/Microsoft.OData.Core/UriParser/Binders/SelectExpandBinder.cs

```diff
@@ -820,34 +847,53 @@ private static LevelsClause ParseLevels(long? levelsOption, IEdmType sourceType,
         }
 
         private static MetadataBinder BuildNewMetadataBinder(ODataUriParserConfiguration config,
+            IEdmNavigationSource navigationSource,
             IEdmNavigationSource targetNavigationSource,

```

Comment:
can you add a comment for this method to illustrate the difference between "navigationSource" and "targetNavigationSource"?

Reply:
I have renamed `navigationSource` to `resourcePathNavigationSource` which makes it clear